lib/lib-grid/src/test/java/org/sagebionetworks/repo/model/grid/node/VectorNodeTest.java

```diff
@@ -127,6 +127,18 @@ public void testAttemptInsertWithNewIndex() {
 				.setValueFromJson("{\"c0\":{\"v\":111,\"i\":[3,4]},\"c1\":{\"v\":222,\"i\":[4,4]}}");
 		assertEquals(expected, vec);
 	}
+	
+	@Test
+	public void testAttemptInsertWithNewNodeIdLesserThanContainerId() {
+		VectorNode vec = new VectorNode().setId(id).setValueFromJson("{\"c0\":{\"v\":111,\"i\":[3,4]}}");
```

Comment:
These JSON strings will need to change with #5421 (depending on which goes in first)